views/login/eligibility-dependents-claim.pug

```diff
@@ -0,0 +1,21 @@
+extends ../base
+
+block variables
+  -var title = __('Tax credits for eligible dependents')
+  -var eligibleDependentsClaim = hasData(data, 'login.eligibleDependentsClaim') ? data.login.eligibleDependentsClaim : ''
+
+block content
+
+  +stepper('Screening questions', '4', '7')
```

Comment:
Since this always says "screening questions", we can just put that in the macro.